sdk/appconfiguration/Azure.Data.AppConfiguration/src/ConfigurationSetting.cs

```diff
@@ -35,19 +34,19 @@ public ConfigurationSetting(string key, string value, string label = null)
         }
 
         /// <summary>
-        /// The primary identifier of a configuration setting.
-        /// The key is used in unison with the label to uniquely identify a configuration setting.
+        /// The primary identifier of the configuration setting.
+        /// The key is used together with label to uniquely identify a configuration setting.
```

Comment:
Here and just below, I would say "a label" or "a key"; or even refer to them via `<see cref=""/>` for a nice tie between the doc pages.